### DISCUSSION TITLE
Bug when querying organization members using GraphQL

### DISCUSSION THREAD

**Discussion by plentz** [community/community#24397] [Apps, API and Webhooks]
<p>Using <a href="https://developer.github.com/v4/explorer/" rel="nofollow noopener">https://developer.github.com/v4/explorer/</a>, I’m able to query all users of my organization with my login. However, if I’m using GraphQL directly using my personal access token (which have all permissions enable), I can only see the public users. This is the query:</p>
<pre><code>query ($organization: String!, $after: String) {
  organization(login: $organization) {
    membersWithRole(first: 100, after: $after) {
      totalCount
      pageInfo {
        hasNextPage
        endCursor
      }
      nodes {
        id
        login
      }
    }
  }
}
</code></pre>

**Comment by plentz** [comment, marked as answer]
<p>Sorry, the problem was that I was using the wrong token when I added the permissions. Solved.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/plentz">@plentz</a>, Thanks for coming back and sharing what worked for you!</p>
